pkg/resources/pullSecret_test.go

```diff
@@ -20,63 +20,6 @@ func getBuildScheme() (*runtime.Scheme, error) {
 	return scheme, err
 }
 
-func TestGetDefaultPullSecret(t *testing.T) {
```

Comment:
Useless tests or broken tests?

Reply:
Useless, they were testing a function  that I've deleted now.